ios/PassCulture.xcodeproj/project.pbxproj

```diff
@@ -42,6 +43,7 @@
 		6D144DC8254C2B760060E0BD /* GoogleService-Info.plist */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = text.plist.xml; path = "GoogleService-Info.plist"; sourceTree = "<group>"; };
 		72D36AF6AC0347509FF395EE /* Montserrat-Bold.ttf */ = {isa = PBXFileReference; explicitFileType = undefined; fileEncoding = 9; includeInIndex = 0; lastKnownFileType = unknown; name = "Montserrat-Bold.ttf"; path = "../assets/fonts/Montserrat/Montserrat-Bold.ttf"; sourceTree = "<group>"; };
 		7365A19A886E4D88A9D9D4C2 /* PassCulture-Bridging-Header.h */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.c.h; path = "PassCulture-Bridging-Header.h"; sourceTree = "<group>"; };
+		82A443CC2AC3693F004BD838 /* Montserrat-BoldItalic.ttf */ = {isa = PBXFileReference; lastKnownFileType = file; name = "Montserrat-BoldItalic.ttf"; path = "../assets/fonts/Montserrat/Montserrat-BoldItalic.ttf"; sourceTree = "<group>"; };
```

Comment:
Pas possible d'avoir le même genre sur Android j'imagine ? :/

Reply:
Sur Android t'as un commit spécifique, c'est plus simple :)

Reply:
Je comprends pas ?

Reply:
Ma question était mal formulée :D Sur Android on est obligés de dupliquer la font. Je me demandais si ce n'était pas possible de la référencer comme on le fait sur iOS

Reply:
J'ai suivi cette [doc](https://www.bam.tech/en/article/add-a-custom-font-to-your-react-native-app) et toutes les précédentes font sont dupliquées donc je pense que c'est nécessaire